### PR TITLE
Add profiles composable

### PR DESCRIPTION
**Pull request by samuveth**
Feel the power of composables :D

### CODE REVIEW

**Comment by bonustrack** [comment]
Looks great! When an user have a 3Box profile and an eth domain, we should show first the 3Box name not the domain, it seem to work everywhere but not on the Space page, can you change that? 
image
Everywhere else:
image

**Comment by bonustrack** [comment]
@samuveth Timeline is broken for me
image
https://bafybeiccwpicpevv4kovymipdj4vx6mvdwrph5khjwyjsdgt4smaiey5te.on.fleek.co/#/explore

**Comment by samuveth** [comment]
@bonustrack Do we also want to load the others like Playground and Delegate?

**Comment by bonustrack** [comment]
@samuveth Don't need for playground but delegate yes.

**Comment by samuveth** [comment]
@bonustrack I'll do the delegate in a separate PR then, as I need to still change it to Comp. API

**Review by bonustrack — APPROVED** [review]
Looks good!

File: src/helpers/utils.ts [hunk 1]
```diff
@@ -7,6 +7,18 @@ export function shorten(str = '') {
   return `${str.slice(0, 6)}...${str.slice(str.length - 4)}`;
 }
 
+export function _shorten(str: string, key?: any): string {
```
Comment (bonustrack) [inline src/helpers/utils.ts:10]: We should reuse this function in the mixins

**Pull request merged**